### PR TITLE
Add futures layout

### PR DESCRIPTION
**Pull request by andytcf**
image

### CODE REVIEW

File: pages/futures/index.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+				</PageContent>
+			</AppLayout>
+		</>
+	);
+};
+
+const StyledRightSideContent = styled(RightSideContent)`
+	padding-left: 32px;
+	padding-right: 32px;
+	> * + * {
```
Comment (clementbalestrat) [inline pages/futures/index.tsx:54]: I'm not a big fan of these CSS selectors as they'd break if anything is added into the HTML code. A bit too generic imo

File: sections/futures/WalletOverview/OverviewRow.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	sign: string;
+};
+
+const OverviewRow: React.FC<OverviewRowProps> = ({ subtitle, data, currencyKey, sign }) => {
+	const { t } = useTranslation();
+	return (
+		<StyledDataRow>
+			<Subtitle>{subtitle}</Subtitle>
+			<FlexDivRowCentered>
+				<CurrencyIcon width={`15px`} currencyKey={currencyKey} />
```
Comment (clementbalestrat) [inline sections/futures/WalletOverview/OverviewRow.tsx:24]: Maybe let's remove `width` from the props and do a `StyledCurrencyIcon` instead

File: sections/futures/WalletOverview/PositionCard.tsx [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+	justify-content: space-between;
+	margin-bottom: 4px;
+`;
+
+const StyledFlexDivRow = styled(FlexDivRow)`
+	justify-content: flex-start;
+	align-items: center;
+`;
+
+const StyledSubtitle = styled(Subtitle)``;
```
Comment (clementbalestrat) [inline sections/futures/WalletOverview/PositionCard.tsx:100]: Probably not needed?

**Review by clementbalestrat — CHANGES_REQUESTED** [review]
Good stuff. A few (minor) comments.

File: pages/futures/index.tsx (hunk 1 above)
Comment (andytcf) [reply pages/futures/index.tsx:54]: removing it, it was copied from shorts

**Pull request merged**